### DISCUSSION TITLE
Determine the top and left padding of an element

### DISCUSSION THREAD

**Discussion by DarkMagus** [clientIO/joint#2678] [Q&A] [answered]
### Introduction

Paperscroller offers a function that lets developer position an element using padding as follow:
paperScroller.positionElement(element, 'top-left', { padding: { top: 50, left: 200 } });

Does JointJS+ offer a function that does the opposite? for example, given an element, the function returns its top and left padding

Thanks

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
This should do:
```ts
element.position().difference(scroller.getVisibleArea().topLeft())
```

**Reply by DarkMagus** [reply]
@kumilingus  Thanks lot
